credoai/modules/constants_metrics.py

```diff
@@ -19,10 +19,13 @@
 
 THRESHOLD_METRIC_CATEGORIES = ["BINARY_CLASSIFICATION_THRESHOLD"]
 
-MODEL_METRIC_CATEGORIES = [
+MODEL_TYPE_METRIC_CATEGORIES = [
```

Comment:
If you think there's gonna be a close correspondence between metric types and model categories, why not just reference the model category constants defined here?`credoai/artifacts/model/constants_model.py`

<img width="1331" alt="image" src="https://user-images.githubusercontent.com/85892367/206558939-66a39c87-fa9e-4af6-9bf9-ada6e74698a5.png">